src/Agent.Service/Windows/Program.cs

```diff
@@ -51,5 +36,57 @@ static int Main(String[] args)
 
             return 0;
         }
+
+        public static void LaunchAgentListener()
+        {
+            AgentListener agentListener = null;
+            while(true)
+            {
+                try
+                {
+                    agentListener = new AgentListener(ExecutionMode.Process);
+                    EventLogger.WriteInfo("Starting VSTS Agent Process");                
```

Comment:
The whole of AgentService code is for windows only and it is built against full .net 

---
In reply to: [115211790](https://github.com/Microsoft/vsts-agent/pull/951#discussion_r115211790) [](ancestors = 115211790)